test/unit/dropin.js

```diff
@@ -186,6 +187,16 @@ describe('Dropin', function () {
       }.bind(this));
     });
 
+    it('injects stylesheet with correct id', function (done) {
```

Comment:
Nope, but we do have [this line of code](https://github.com/braintree/braintree-web-drop-in/blob/9e9e9e61ba48cf5676bfb751ab6356a6ef1ea57b/src/dropin.js#L147)